test/user.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require('supertest');
+
+const app = require('../src/backend/web/app');
+
+jest.mock('../src/backend/utils/elastic');
+
+// Mock the internal authentication strategy
+jest.mock('../src/backend/web/authentication');
+// Use our authentication test helper
+const { login, logout } = require('./lib/authentication');
+
+describe.skip('test GET /user/feeds endpoint', () => {
+  let user;
+
+  beforeAll(() => {
+    user = login('Johannes Kepler', 'user1@example.com');
```

Comment:
Why bother doing this if you're going to always logout on 20?

Reply:
I did this to ensure all test functions can access reference user1's information (via the `user` variable on L13).

If I removed this `beforeAll()` call, I still would have to call `login()` within the last test function (even though the user is already logged in) in order to get the user information I need to build [the `feedData` object](https://github.com/Seneca-CDOT/telescope/pull/958/files#diff-a202d1e6aac601eca4c15ad2418135fdR42).

Reply:
I did this to ensure all test functions can access reference user1's information (via the `user` variable on L13).

If I removed this `beforeAll()` call, I still would have to call `login()` within the last test function (even though the user is already logged in) in order to get the user information I need to build [the `feedData` object](https://github.com/Seneca-CDOT/telescope/pull/958/files#diff-a202d1e6aac601eca4c15ad2418135fdR42).